packages/grafana-data/src/types/panel.ts

```diff
@@ -27,6 +27,12 @@ export interface PanelData {
   /** Contains data frames with field overrides applied */
   series: DataFrame[];
 
+  /** A list of annotation items */
+  annotations?: DataFrame[];
+
+  /** A list of exemplar data */
+  exemplars?: DataFrame[];
```

Comment:
this makes me really want typed data frames :) 

like 
https://github.com/data-forge/data-forge-ts/blob/master/src/lib/dataframe.ts#L279

Reply:
but why?  don't exemplars include the same variation in data?   I am assuming this is a method to say "the query is aggregating a bunch of data, but here are a few samples of real data" -- or am i thinking about it wrong?

Do exemplars have a strict structure?

Reply:
They need some assumed structure. But your right we can be quite flexible and use the first date field, first number for value, first string for traceId.

Reply:
Right now for example we have some runtime functions to check for example if dataframe is timeseries, which I think is implemented as has a time column and at least one numeric column. Having that typed would probably help quite a bit in general.